scripts/compile_tests.js

```diff
@@ -99,4 +99,22 @@ if (!fs.existsSync(publisherPath) && !fs.existsSync(subscriptionPath) &&
 } else {
   copyAll([publisherPath, subscriptionPath, , listenerPath, clientPath], testCppDir);
 }
+
+let actionPath = path.join(path.dirname(__dirname), 'test', 'ros1_actions');
+let actionBuildProcess = child.spawn('colcon', ['build', '--base-paths', actionPath]);
+actionBuildProcess.on('close', (code) => {
+  if (os.platform() === 'win32') {
+    child.spawn('cmd.exe',
+      ['/c', `xcopy ${path.join(rootDir, 'install', 'ros1_actions')} ${process.env.AMENT_PREFIX_PATH} /O /X /E /K`]);
+  } else {
+    child.spawn('sh',
+      ['-c', `cp -r ${path.join(rootDir, 'install', 'ros1_actions') + '/.'} ${process.env.AMENT_PREFIX_PATH}`]);
+  }
+});
+actionBuildProcess.stdout.on('data', (data) => {
+  console.log(`${data}`);
+});
+actionBuildProcess.stderr.on('data', (data) => {
+  console.log(`${data}`);
+});
```

Comment:
Would you please abstract and wrap a function which can build and install a existing packages. So the `rclnodejs_test_msgs` can share the function with actionlib.

Reply:
I'll do that.